language-server/modules/langserver-core/src/test/resources/codeaction/extract-to-local-variable/config/extractToVariableInMappingConstructor4.json

```diff
@@ -0,0 +1,48 @@
+{
+  "range": {
+    "start": {
+      "line": 3,
+      "character": 30
+    },
+    "end": {
+      "line": 3,
+      "character": 62
+    }
+  },
+  "source": "extractToVariableInMappingConstructor2.bal",
```

Comment:
We can use the same source file without duplicating it,  as the source has not changed.

Reply:
Is it okay to use sources from different files? extract to function and local var?

Reply:
I just noticed that. Please ignore the comment